lib/src/utils/schema/actions/sections.spec.ts

```diff
@@ -9,7 +9,7 @@ describe('section actions', () => {
     const defaultSectionName = 'testSection';
 
     // Add a section after the initial one
-    let newSchema = addSection(defaultSchema, `root_${defaultSectionName}`, Section);
+    let newSchema = addSection(defaultSchema, `root_${defaultSectionName}`, Section, undefined);
```

Comment:
From the addSection signature, the last argument is optional so I do not understand why you put `undefined` here.